ccp/modules/id-management-compose.yml

```diff
@@ -101,5 +101,12 @@ services:
       forward_proxy:
         condition: service_healthy
 
+  ccp-patient-project-identificator:
+    image: samply/ccp-patient-project-identificator
+    container_name: bridgehead-ccp-patient-project-identificator
+    environment:
+      MAINZELLISTE_APIKEY: ${IDMANAGER_LOCAL_PATIENTLIST_APIKEY}
+      SITE_NAME: ${SITE_NAME}
+
```

Comment:
Is this supposed to be in this PR?

Reply:
I would appreciate if this could make it into main with this PR. But we can move it to the next merge. It would add the PPI and it should not break anything.

Reply:
I would like it to be in a different PR for transparency reasons. However, we could merge them at the same-ish time.

Reply:
```suggestion
```
agreed, move to different PR